### PR TITLE
Gs nest integration

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
## Proposed changs
- [ ] Authentication
- [ ] GET endpoints for all resources
- [ ] PUT connection

### CODE REVIEW

File: src/pages/ConnectionRequests/index.tsx [hunk 1]
```diff
@@ -30,11 +33,11 @@ const ConnectionRequestsContainer: React.FC<PropsFromRedux> = ({
   requests,
 }) => {
   const handleAccept = (request: BaseConnection): void => {
-    updateConnection({ connectionId: request.id, status: "approved" });
+    updateConnection({ connectionId: request.id, status: "active" });
```
Comment (lspdrz) [inline src/pages/ConnectionRequests/index.tsx:36]: Nice. Can we put these values in an enum?

File: src/pages/Dashboard/index.tsx [hunk 2]
```diff
@@ -108,7 +108,7 @@ function Dashboard({
                   calls.filter(
                     (call) =>
                       call.status === "live" ||
-                      call.status === "missing-monitor"
+                      call.status === "missing_monitor"
```
Comment (lspdrz) [inline src/pages/Dashboard/index.tsx:111]: Same here, can we place in enum?

File: src/pages/Inmates/index.tsx [hunk 3]
```diff
@@ -58,6 +56,21 @@ const UnconnectedInmateContainer: React.FC<PropsFromRedux> = ({
       dataIndex: "location",
       editable: true,
     },
+    {
+      title: "",
+      key: "acction",
```
Comment (lspdrz) [inline src/pages/Inmates/index.tsx:61]: typo?

File: src/pages/LiveCall/index.tsx [hunk 4]
```diff
@@ -78,13 +71,14 @@ const LiveVisitationContainer: React.FC<PropsFromRedux> = ({
   }, [setSocket, socket]);
 
   useEffect(() => {
-    // setVisibleCalls(visitations.slice(0, grid));
+    console.log("running");
```
Comment (lspdrz) [inline src/pages/LiveCall/index.tsx:74]: remove?

File: src/pages/LiveCall/index.tsx [hunk 5]
```diff
@@ -78,13 +71,14 @@ const LiveVisitationContainer: React.FC<PropsFromRedux> = ({
...
     if (!activeCallChatId && visitations.length > 0) {
       setActiveCallChatId(visitations[0].id);
     }
   }, [grid, visitations, activeCallChatId]);
 
   useEffect(() => {
+    console.log("rernrender");
```
Comment (lspdrz) [inline src/pages/LiveCall/index.tsx:81]: remove?

File: src/typings/Connection.ts [hunk 6]
```diff
@@ -1,4 +1,7 @@
-export type ConnectionStatus = "approved" | "denied" | "pending";
+import { Contact } from "./Contact";
+import { Inmate } from "./Inmate";
+
+export type ConnectionStatus = "active" | "pending" | "inactive" | "rejected";
```
Comment (lspdrz) [inline src/typings/Connection.ts:4]: yeah, can we use these values in the codebase?

**Review by lspdrz — APPROVED** [review]

File: src/pages/ConnectionRequests/index.tsx (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/pages/ConnectionRequests/index.tsx:36]: we're enforcing string literal types
personally, I am pro string literal types over enums client-side.

**Pull request merged**